### DISCUSSION TITLE
radial progress animation

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [saadeghi/daisyui#1585] [Q&A] [answered]
Hi,

Quick question.

How can i make circle animation for the radial progress bar .

<div class="radial-progress" style="--value:80;">80%</div>

**Comment by saadeghi** [comment, marked as answer]
It works with CSS conic gradient. We can't animate the degree of a conic gradient using CSS, unless `@property` is supported by all browsers in the future.  
For now, you can use JS to increase the number from `0` to `80` (or anything) and it will work.

Let me know if you have any questions.

**Reply by MMore** [reply]
Ah I was about to ask the same question...
In my case I am using Phoenix LiveView to morph the DOM in the end. So the JS approach is not working here. Any other ideas?

**Reply by saadeghi** [reply]
After a year, now all browsers support `@property` except Firefox (will support in version 124 soon)  
So I think it's now safe to use. I will work on it.
